src/test/java/br/com/openenade/api/AnoUnityTests.java

```diff
@@ -0,0 +1,99 @@
+package br.com.openenade.api;
+
+import static org.junit.Assert.*;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.transaction.annotation.Transactional;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@Transactional
+public class AnoUnityTests {
+
+    @Autowired
+    private AnoRepository repository;
```

Comment:
I think that testing the AnoService instead of the AnoRepository will improve the reliability of the API :).